Changelog.skroutz.md

```diff
@@ -0,0 +1,79 @@
+## 0.4.0.skroutz.0 (unreleased)
```

Comment:
I'd suggest to make this 0.3.0.skroutz.0 instead, so that it's apparent when exactly we forked off of the upstream code.

Otherwise, we should at least state this with a comment right before we state the changes of this tag.

Reply:
Ok, currently the version running in yogurt is `0.3.0.1.skroutz`. So i am not sure what is the version we want.

Reply:
Also from the upstream commit https://github.com/nevans/resque-pool/commit/61ba598f6c78f5350bb1d716694188ca6618417b at the time we forked, the last tag is `0.4.0.rc1` so IMHO `0.4.0.skroutz.0` should be fine.